### PR TITLE
Fix to fit #557 into our workflow.

### PR DESCRIPTION
**Pull request by hayajo**
Ref #557.
Thank you, @tnishinaga

- Build arm64 and mips binary at the crossbuild-package task
- Build deb packeages for arm64 and mips at the dev-v2 task
- Build deb packages in Dockerlized environment

### CODE REVIEW

File: packaging/deb-systemd/debian/control [hunk 1]
```diff
@@ -9,7 +9,7 @@ Vcs-Git: https://github.com/mackerelio/mackerel-agent.git
 Homepage: https://mackerel.io
 
 Package: mackerel-agent
-Architecture: amd64
+Architecture: any
```
Comment (astj) [inline packaging/deb-systemd/debian/control]: ref: https://www.debian.org/doc/debian-policy/ch-controlfields.html#architecture

When `any`, produced binary package will have specific architecture (amd64, mips, etc…), which is desired behavior.

**Comment by astj** [comment]
@hayajo (and @tnishinaga ) This p-r and #557 will build only systemd-based deb package for arm and mips. I hope this will satisfy major use cases, but in some cases it might not fit. For example, if you want to install mackerel-agent into some limited environment which does not use systemd, like NAS/router with busybox, this deb will not work since they works on top of systemd.
Do you think current artifacts are enough for use cases?

File: .gitignore [hunk 2]
```diff
@@ -18,5 +18,10 @@ packaging/deb/debian/mackerel-agent.bin
 packaging/*.tar.gz
 packaging/deb-build
 packaging/rpm-build
+packaging/*.deb
```
Comment (astj) [inline .gitignore]: Hmm, should we change directory structure to be more gitignore-friendly?

File: .travis.yml [hunk 3]
```diff
@@ -9,13 +9,6 @@ env:
   - PATH=/home/travis/gopath/bin:$PATH DEBIAN_FRONTEND=noninteractive
   - secure: "VG3xbEjFSuB80QBFchHZqQDB4VF74RTXaNSBPl/bavGr9My2RSqtfXqfEGUHFwo5zPM7LA0bIEC7jsobf6M6vtfVgJKOOXEfF/pfD83n++eHZe0yLN6Wysp1Bn14KH+ont550gDityORqEvX3pK4k9hSb8dXpjGnCtrAwqAS+eg="
   - GO111MODULE=on
-addons:
```
Comment (astj) [inline .travis.yml]: 😄

File: .gitignore (hunk 2 above)
Comment (hayajo) [reply .gitignore]: It will be nice to put the artifacts in one directory.
But we also need to fix other tools, so let's discuss in another issue.

**Pull request merged**